common/get_guest_system_info.yml

```diff
@@ -45,3 +45,15 @@
     guest_os_family: "{{ guest_system_info.ansible_os_family | default('') }}"
     guest_os_product_type: "{{ guest_system_info.ansible_os_product_type | default('') }}"
     guest_os_hostname: "{{ guest_system_info.ansible_hostname | default('') }}"
+
+- name: "Set fact of ansible_pkg_mgr manually when it'is unknown"
```

Comment:
Please update the task name to "Set fact of package manager for {{ guest_os_ansible_distribution }}". Do not update `ansible_pkg_mgr` please. But you can also update `guest_os_ansible_distribution_ver` and `guest_os_ansible_distribution_major_ver` in case there is version comparison in future.
```
guest_os_ansible_distribution_ver: "{{ guest_os_ansible_distribution_ver  | regex_replace('V', '', ignorecase=True) }}"
guest_os_ansible_distribution_major_ver:  "{{ guest_os_ansible_distribution_major_ver | regex_replace('V', '', ignorecase=True) | int }}"
```